src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,66 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import * as React from 'react';
+import { TestId } from '../../utils/types';
+import { backgroundContainer, cardSizing, closeContainer, modalContainer } from './Modal.style';
+import { generateTestDataId } from '../../utils/helpers';
+import Card from '../Card';
+import IconButton from '../IconButton';
+import ClickAwayListener from '../utils/ClickAwayListener';
+import ModalContent, { Props as ModalContentProps } from './ModalContent/ModalContent';
+
+export type Props = {
+  /**  If true, the modal is open. Defaults to false. */
+  open: boolean;
+  /** Callback fired when the component requests to be closed. */
+  onClose: () => void;
+  /** If true, modal will use default ModalContent and contentProps should be ModalContentProps. Defaults to false */
+  fixedContent?: boolean;
```

Comment:
I think you can remove this and only use the contentProps but it's optional, so I will leave it to you.